src/main/resources/templates/login.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Login</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+</head>
+
+<body>
+    <div class="container">
+        <h1 id="title">Login</h1>
+                <div>
+                    <p id="username"><b>Username:</b></p>
+                    <p id="password"><b>Password:</b></p>
+                </div>
+                
+                <div>
+                	<button class="btn btn-primary">Sign-in</button>
+                	<button class="btn btn-primary">Forgot Password</button>
+                </div>
+        </div>
+    <script src="js/index.js"></script>
+</body>
+
+</html>
```

Comment:
Created the page that the application will route to for when a user logs in. Setup based on how the page is laid out on your user story within the design document for your group.